### PR TITLE
Refine Readme.md for 1.3 release

### PR DESCRIPTION
**Pull request by scarlett2018**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,19 +108,31 @@
           </ul>
         </ul>
           <a href="docs/en_US/NAS/Overview.md">Neural Architecture Search</a>
-          <ul>
-          <li><a href="docs/en_US/NAS/Overview.md#enas">ENAS</a></li>
-          <li><a href="docs/en_US/NAS/Overview.md#darts">DARTS</a></li>
-          <li><a href="docs/en_US/NAS/Overview.md#p-darts">P-DARTS</a></li>
-          <li><a href="docs/en_US/Tuner/BuiltinTuner.md#NetworkMorphism">Network Morphism</a> </li>
+          <ul>            
+            <b>One-Shot</b>
```
Comment (xuehui1991) [inline README.md]: Most of people don't think they are one-shot models.....

Comment (scarlett2018) [reply README.md]: The current NAS document categorize these as one-shot, let's discuss about this offline... maybe we don't need this sub-category here.

**Comment by scarlett2018** [comment]
Table refactor option for discussion:

image

File: README.md (hunk 1 above)
Comment (scarlett2018) [reply README.md]: I simply removed the NAS category for now.

**Comment by scarlett2018** [comment]
will need @yangou1988 to update the overview image per offline discussion.

**Comment by yangou1988** [comment]
overview_v2_画板 1 副本 2

overview image updated.

**Pull request merged**